Sources/HaCWebsiteLib/ViewModels/NotFound.swift

```diff
@@ -0,0 +1,27 @@
+import HaCTML
+
+struct NotFound {
+  
+  var node: Node {
+    return UI.Pages.base(
+      title: "Page Not Found",
+      content: Fragment(
+        El.Div[Attr.className => "NotFound_container"].containing(
+          El.Img[
+            Attr.className => "LandingIntroduction__titleImage",
```

Comment:
So we're using a bunch of classes from the `LandingIntroduction` block here. Our conventions specify that they don't make sense outside of the context of their containing block, and may break at any time. Semantically, this is also questionable.

Essentially, people would eventually change (or even remove) this `LandingIntroduction` block, and it won't be obvious that it will affect our 404 page as well, possibly breaking it.

We have some options to get around this:

- Create elements specific to `NotFound` (e.g. `NotFound__logoImage`); _or_
- Abstract these elements into blocks of their own so they can actually be re-usable components.

For example, we could break this logo + "Cambridge's student tech society" thing into it's own block, `SiteLogo`, with a corresponding view model so we can reuse it.